tests/phpunit/tests/auth.php

```diff
@@ -1591,6 +1614,17 @@ public function test_application_password_authentication() {
 		$this->assertSame( $item['uuid'], rest_get_authenticated_app_password() );
 	}
 
+	/**
+	 * @ticket 21022
+	 * @ticket 63203
+	 */
+	public function test_application_password_is_hashed_with_fast_hash() {
+		// Create a new app-only password.
+		list( $_, $item ) = WP_Application_Passwords::create_new_application_password( self::$user_id, array( 'name' => 'phpunit' ) );
```

Comment:
nit: `$_` isn't used. It can be left out when calling list.  https://3v4l.org/8DTrj
```suggestion
		list( , $item ) = WP_Application_Passwords::create_new_application_password( self::$user_id, array( 'name' => 'phpunit' ) );
```